packages/cubejs-schema-compiler/adapter/BaseFilter.js

```diff
@@ -79,8 +79,15 @@ class BaseFilter extends BaseDimension {
     return this.operator === 'contains' || this.operator === 'not_contains';
   }
 
+  isDateOperator() {
+    // Fix camelCase => snake_case
+    const operator = this.operator.replace(/[A-Z]/g, letter => `_${letter.toLowerCase()}`);
+
+    return contains(operator, DATE_OPERATORS);
+  }
+
   filterParams() {
-    if (contains(this.operator, DATE_OPERATORS)) {
+    if (this.isDateOperator()) {
```

Comment:
This place is handled in api-gateway. To make this change I'd suggest separate PR and update all the tests so we can get rid of the snake case.

Reply:
It's really better to separate, please review: https://github.com/cube-js/cube.js/pull/999